lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -380,16 +382,16 @@ newStakePoolLayer tr block0H getEpCst db@DBLayer{..} nl metadataDir = StakePoolL
     computeProgress
         :: BlockHeader -- ^ ... / denominator
         -> BlockHeader -- ^ numerator /...
-        -> Quantity "percent" Percentage
+        -> Ratio Word64
```

Comment:
I guess I should keep returning a `Quantity "percent" Percentage` here by using `unsafeMkPercentage`